### PR TITLE
Feature/calimages datecrt field

### PR DESCRIPTION
**Pull request by turnerm**

### CODE REVIEW

File: banzai/utils/fits_utils.py [hunk 1]
```diff
@@ -45,6 +46,7 @@ def create_master_calibration_header(images):
     mean_dateobs = date_utils.mean_date(observation_dates)
 
     header['DATE-OBS'] = (date_utils.date_obs_to_string(mean_dateobs), '[UTC] Mean observation start time')
+    header['DATE-CRT'] = (date_utils.date_obs_to_string(datetime.datetime.utcnow()), '[UTC] Master creation time')
```
Comment (cmccully) [inline banzai/utils/fits_utils.py]: Often, the convention is that DATE is the date the fits file was created.

File: banzai/dbs.py [hunk 2]
```diff
@@ -67,6 +67,7 @@ class CalibrationImage(Base):
     filename = Column(String(50), unique=True)
     filepath = Column(String(100))
     dateobs = Column(DateTime, index=True)
+    datecrt = Column(DateTime, index=True)
```
Comment (cmccully) [inline banzai/dbs.py]: I dislike datecrt. Why not datecreated?

File: banzai/images.py [hunk 3]
```diff
@@ -166,6 +168,9 @@ def write_catalog(self, filename, nsources=None):
     def add_history(self, msg):
         self.header.add_history(msg)
 
+    def upate_datecrt(self):
```
Comment (cmccully) [inline banzai/images.py]: Why does this need its own method?

File: banzai/utils/fits_utils.py (hunk 1 above)
Comment (turnerm) [reply banzai/utils/fits_utils.py]: I switched to using `DATE`

File: banzai/dbs.py (hunk 2 above)
Comment (turnerm) [reply banzai/dbs.py]: Changed to `datecreated`

File: banzai/images.py (hunk 3 above)
Comment (turnerm) [reply banzai/images.py]: I wanted to avoid directly modifying an object attribute.

**Pull request merged**